plugins/GSdx/GSHwHack.cpp

```diff
@@ -752,6 +752,7 @@ bool GSC_TalesOfLegendia(const GSFrameInfo& fi, int& skip)
 
 bool GSC_Kunoichi(const GSFrameInfo& fi, int& skip)
 {
+	// Removes depth effects(shadows) not rendered correctly on all renders.
```

Comment:
This one as well in it's own skip value.

Reply:
I can not understand you. Unfortunately I do not know English well.

Reply:
Move the comment where skip line is, above it.
 		{
 			// Removes depth effects(shadows) not rendered correctly on all renders.
 			skip = 1;
 		}